### PR TITLE
test that preview is working and making lighting changes

### PR DESCRIPTION
**Pull request by kotoole1**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357755%22%2C%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23issuecomment-87461090%22%2C%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357907%22%2C%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357909%22%2C%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357914%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23issuecomment-87461090%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Fix%20yo%20branch%22%2C%20%22created_at%22%3A%20%222015-03-29T19%3A32%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1248454%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/lukemetz%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%209e34f8e71212853ceefaab04fc699503c1bbdfcc%20Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357907%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20we%20move%20this%20to%20a%20helper%20class%20of%20somekind%3F%22%2C%20%22created_at%22%3A%20%222015-03-29T19%3A43%3A48Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2672344%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kotoole1%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift%3AL1-40%22%7D%2C%20%22Pull%209e34f8e71212853ceefaab04fc699503c1bbdfcc%20Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift%2022%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357909%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Left%20a%20fit%20in.%22%2C%20%22created_at%22%3A%20%222015-03-29T19%3A44%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2672344%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kotoole1%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift%3AL1-40%22%7D%2C%20%22Pull%2061ed2f8d83d46907bf063b13c0a1a1ac064c7854%20Paranormal/Paranormal/GPUImageFilters/BlendFilters/BlendSmoothFilter.swift%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357755%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Bad%20indent.%22%2C%20%22created_at%22%3A%20%222015-03-29T19%3A30%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1248454%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/lukemetz%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20Paranormal/Paranormal/GPUImageFilters/BlendFilters/BlendSmoothFilter.swift%3AL26-33%22%7D%2C%20%22Pull%209e34f8e71212853ceefaab04fc699503c1bbdfcc%20Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift%2028%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/lukemetz/Paranormal/pull/99%23discussion_r27357914%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Left%20a%20write%22%2C%20%22created_at%22%3A%20%222015-03-29T19%3A44%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2672344%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kotoole1%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift%3AL1-40%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 61ed2f8d83d46907bf063b13c0a1a1ac064c7854 Paranormal/Paranormal/GPUImageFilters/BlendFilters/BlendSmoothFilter.swift 5'></a> &nbsp;<b><a href='https://github.com/lukemetz/Paranormal/pull/99#discussion_r27357755'>File: Paranormal/Paranormal/GPUImageFilters/BlendFilters/BlendSmoothFilter.swift:L26-33</a></b>
- <a href='https://github.com/lukemetz'></a> Bad indent.
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/lukemetz/Paranormal/pull/99#issuecomment-87461090'>General Comment</a></b>
- <a href='https://github.com/lukemetz'></a> Fix yo branch
- [ ] <a href='#crh-comment-Pull 9e34f8e71212853ceefaab04fc699503c1bbdfcc Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift 16'></a> &nbsp;<b><a href='https://github.com/lukemetz/Paranormal/pull/99#discussion_r27357907'>File: Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift:L1-40</a></b>
- <a href='https://github.com/kotoole1'></a> Can we move this to a helper class of somekind?
- [x] <a href='#crh-comment-Pull 9e34f8e71212853ceefaab04fc699503c1bbdfcc Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift 22'></a> &nbsp;<b><a href='https://github.com/lukemetz/Paranormal/pull/99#discussion_r27357909'>File: Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift:L1-40</a></b>
- <a href='https://github.com/kotoole1'></a> Left a fit in.
- [x] <a href='#crh-comment-Pull 9e34f8e71212853ceefaab04fc699503c1bbdfcc Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift 28'></a> &nbsp;<b><a href='https://github.com/lukemetz/Paranormal/pull/99#discussion_r27357914'>File: Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift:L1-40</a></b>
- <a href='https://github.com/kotoole1'></a> Left a write

<a href='https://www.codereviewhub.com/lukemetz/Paranormal/pull/99?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/lukemetz/Paranormal/pull/99?approve=1'></a>&nbsp;<a href='https://github.com/lukemetz/Paranormal/pull/99'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: Paranormal/Paranormal/GPUImageFilters/BlendFilters/BlendSmoothFilter.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ class BlendSmoothFilter : BlendFilter {
 
         initialFilters = [replaceAlphaFilter, blurFilter]
         terminalFilter = blendAddFilter
-    }
+}
```
Comment (lukemetz) [inline Paranormal/Paranormal/GPUImageFilters/BlendFilters/BlendSmoothFilter.swift]: Bad indent.

**Comment by lukemetz** [comment]
Fix yo branch

File: Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+class previewTests: QuickSpec {
+    override func spec() {
+        describe("Preview on new document") {
+            var previewController : PreviewViewController!
+            var documentController : DocumentController!
+
+            beforeEach {
+                documentController = DocumentController()
+                previewController = PreviewViewController()
+                for doc in documentController.documents {
```
Comment (kotoole1) [inline Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift]: Can we move this to a helper class of somekind?

File: Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                    documentController.removeDocument(doc as NSDocument)
+                }
+                documentController.createDocumentFromUrl(nil)
+            }
+
+            fit("Renders a preview") {
```
Comment (kotoole1) [inline Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift]: Left a fit in.

File: Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                expect(documentController.documents.count).to(equal(1))
+                let document = documentController.documents[0] as? Document
+                PNEditorHelper.waitForEditorImageInDocument(document!)
+                expect(document?.computedPreviewImage).toEventuallyNot(beNil())
+                if let image = document?.computedPreviewImage {
+                    NSImageHelper.writeToFile(image, path: "/Users/kotoole/Desktop")
```
Comment (kotoole1) [inline Paranormal/ParanormalTests/Panels/Preview/PreviewTests.swift]: Left a write

**Pull request merged**